config/default/csp.settings.yml

```
@@ -0,0 +1,64 @@
+report-only:
+  enable: false
+  directives: {  }
```

YAMLlint (1.38.0)

[error] too many spaces inside empty braces

(braces)